talk/owt/sdk/include/cpp/owt/base/connectionstats.h

```diff
@@ -4,13 +4,950 @@
 #ifndef OWT_BASE_CONNECTIONSTATS_H_
 #define OWT_BASE_CONNECTIONSTATS_H_
 #include <chrono>
+#include <map>
 #include <memory>
 #include <string>
 #include <vector>
 #include "owt/base/commontypes.h"
 #include "owt/base/network.h"
 namespace owt {
 namespace base {
+
+
+// RTC-spec compliant statistics interfaces.
```

Comment:
If you want descriptions to be included in documentation, please follow doxygen's syntax.

Reply:
done. Only those we would like to expose at present is commented with \sa.